sample-shrinker.sh

```diff
@@ -251,6 +255,32 @@ prep_bitdepth_convert()
 }
 
 
+# Determine what do to about sample rate
+# - updates $change_summary with a text summary
+# - updates $*_args with sox args
+prep_samplerate_convert()
+{
+  # Prepare samplerate conversion
+  if [[ $samplerate -ne $target_samplerate ]]; then
+    change_summary+="${samplerate}      "
+    if [[ $samplerate -gt $target_samplerate ]]; then
+      change_summary+="${samplerate}->${target_samplerate}"
+      dst_args+=(--rate="$target_samplerate")
+
+    # Raise below-minimum samplerate samples to minimum samplerate (default: 11025)
+    elif (( samplerate < target_samplerate && samplerate < 11025 )); then
```

Comment:
```suggestion
    elif (( samplerate < target_samplerate && samplerate < ${minimum_samplerate:-11025} )); then
```

Reply:
This minimum stuff is what I was most unsure about - feel free to continue to push in the right direction or rip it all out if I got it wrong

Reply:
Actually... I notice bit depth has the same problem (probably why this does, too). :smile:    They can both wait until a later update